01-node-tutorial/answers/10-fs-sync.js

```diff
@@ -0,0 +1,6 @@
+const { readFileSync, writeFileSync } = require('fs')
+
+writeFileSync('./temporary/fileA.txt', `Hello World!\n`, { flag: 'a' })
```

Comment:
You need to write 3 lines. You can do that by repeating the command 2 more times with different contents.

Reply:
Resolved.